dev/http_test.go

```diff
@@ -0,0 +1,51 @@
+package dev
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var testHttp HTTPServer
+
+func TestHttp_removeTLD_test(t *testing.T) {
+	str := testHttp.removeTLD("psychic-octo-guide.test")
+
+	assert.Equal(t, "psychic-octo-guide", str)
+}
+
+func TestHttp_removeTLD_noTld(t *testing.T) {
+	str := testHttp.removeTLD("shiny-train")
+
+	assert.Equal(t, "shiny-train", str)
+}
+
+func TestHttp_removeTLD_mutlipartDomain(t *testing.T) {
+	str := testHttp.removeTLD("expert-eureka.loc.al")
+
+	assert.Equal(t, "expert-eureka.loc", str)
```

Comment:
This is to document desired, but somewhat surprising behavior.

See https://github.com/puma/puma-dev/issues/204#issuecomment-523464625